keras_rcnn/layers/pooling.py

```diff
@@ -55,3 +57,48 @@ def call(self, x, **kwargs):
 
     def compute_output_shape(self, input_shape):
         return None, self.regions, self.size, self.size, self.channels
+
+
+class ROIAlign(ROI):
```

Comment:
I like your implementation. It reminds me of the G-RMI implementation from Jonathan Huang, et al.